resources/views/projects/create.blade.php

```diff
@@ -101,44 +93,53 @@
 		<!-- Milestone -->
 		<div class="field {{ $errors->has('milestone') ? 'error': '' }}">
 			<label for="milestone">Public milestone</label>
-			<input type="text" name="milestone" value="{{ old('milestone') }}" placeholder="e.g. Design">
+			<input type="text" id="milestone" value="{{ old('milestone') }}" placeholder="e.g. Design">
```

Comment:
Si no le das un `name` a esto, no me lo va a pasar al back-end.

Reply:
sorry - lo borré por accidente poniendo el `id` para los `for` attr de los labels. 😞 it's there now!